test/e2e/storage/drivers/in_tree.go

```diff
@@ -1148,6 +1148,7 @@ func InitGcePdDriver() testsuites.TestDriver {
 				testsuites.CapFsGroup:     true,
 				testsuites.CapBlock:       true,
 				testsuites.CapExec:        true,
+				testsuites.CapExpansion:   true,
```

Comment:
We can perhaps fix this in a follow up but there would be volumes that only require control plane expansion and volume types that require both and hence this capability does not expresses that intent very well.

For example - if we were to enable these tests for glusterfs driver, there won't be a file system resize happening and tests may need some tweaking.

Reply:
I'm implementing controller side code of csi-driver-nfs [here](https://github.com/kubernetes-csi/csi-driver-nfs/pull/13), and this driver also won't require controller expansion implementation.

In that case, could we just set true for such driver's capability for the test code, or do we need to add special handling for it?

Reply:
I think this is fine for now. Once we start adding tests for driver which need to skip control or node expansion, we can revisit this.

Reply:
Now that we run the testsuite outside of kubernetes, these capabilities are becoming more of an external API, and we should not try to change them too much.  Can we pre add NodeExpansion and ControllerExpansion capabilities now to anticipate the future tests?

Reply:
Separated `NodeExpansion` and `ControllerExpansion` capabilities. I guess that only drivers that have `NodeExpansion` should transit their PVC conditions, so I changed the test code as so. Please check if it is correct.